doc/loggingAndErrorReporting.md

```diff
@@ -65,6 +67,10 @@ For example, in the above example, the log of Jan 1, 2013 gets written to `myapp
 The syntax of the autoRollAtTime is specified by [TimeSpan.ToString()](https://msdn.microsoft.com/en-us/library/1ecy8h51(v=vs.110).aspx).
 For example, in the above example, at the start of the day it will roll the file over.
 
+The zipolderthannumdays can only be used in conjection with autoRollAtTime, provide the number of days of files to keep.
```

Comment:
I would split it to a separate documentation section so that it becomes explicit that the vars are optional. 
I do not see specific reason why it's not supported in other rotating loggers, but fine with me if documented separately

Reply:
Done